src/main/webapp/availability.html

```diff
@@ -43,506 +43,11 @@ <h5 class="text-center lead">Update your availability to be an Interviewer for t
-          </div>
-          <p class="form-labels">:</p>
-          <div class="col">
-            <select class="form-control">
-              <option>00</option>
-              <option>15</option>
-              <option>30</option>
-              <option>45</option>
-            </select>
-          </div>
-          <div class="col">
-            <select class="form-control">
-              <option>AM</option>
-              <option>PM</option>
-            </select>
-          </div>
-        </div>
-      </form>
-      <br>
-      
-      <h5 class="font-weight-bold">Saturday 7/4</h5>
-      <form>
-        <div class="form-row">
-          <p class="form-labels">Start Time:</p>
-          <div class="col">
-            <select class="form-control">
-              <option>1</option>
-              <option>2</option>
-              <option>3</option>
-              <option>4</option>
-              <option>5</option>
-              <option>6</option>
-              <option>7</option>
-              <option>8</option>
-              <option>9</option>
-              <option>10</option>
-              <option>11</option>
-              <option>12</option>
-            </select>
-          </div>
-          <p class="form-labels">:</p>
-          <div class="col">
-            <select class="form-control">
-              <option>00</option>
-              <option>15</option>
-              <option>30</option>
-              <option>45</option>
-            </select>
-          </div>
-          <div class="col">
-            <select class="form-control">
-              <option>AM</option>
-              <option>PM</option>
-            </select>
-          </div>
-          <p class="form-labels end-time">End Time:</p>
-          <div class="col">
-            <select class="form-control">
-              <option>1</option>
-              <option>2</option>
-              <option>3</option>
-              <option>4</option>
-              <option>5</option>
-              <option>6</option>
-              <option>7</option>
-              <option>8</option>
-              <option>9</option>
-              <option>10</option>
-              <option>11</option>
-              <option>12</option>
-            </select>
-          </div>
-          <p class="form-labels">:</p>
-          <div class="col">
-            <select class="form-control">
-              <option>00</option>
-              <option>15</option>
-              <option>30</option>
-              <option>45</option>
-            </select>
-          </div>
-          <div class="col">
-            <select class="form-control">
-              <option>AM</option>
-              <option>PM</option>
-            </select>
-          </div>
-        </div>
-      </form>
-      <br>
-
-      <button type="submit" class="btn btn-primary mb-2 submit-button">Update Availability</button>
+         time range per day (have a plus button)--> 
+      <div id="table-container"></div>
```

Comment:
Consider putting a comment here explaining how this container is populated.

Reply:
Done